DigitalLearningSolutions.Web/Services/RegistrationService.cs

```diff
@@ -398,7 +439,7 @@ public void PromoteDelegateToAdmin(AdminRoles adminRoles, int? categoryId, int u
                     true
                 );
 
-                registrationDataService.RegisterAdmin(adminRegistrationModel);
+                registrationDataService.RegisterAdmin(adminRegistrationModel, new PossibleEmailUpdate());

```

Comment:
Might be better for `RegisterAdmin` (and subsequent functions) to take a `PossibleEmailUpdate? = null`, so that this isn't necessary